src/Component/MovieCard/MovieCards.js

```diff
@@ -1,10 +1,13 @@
 import './MovieCards.css'
 
 
-function MovieCards({ movie }){
+function MovieCards(props){
+    const {movie} = props
     return (
       <div className="movie-cards">
         <img src={movie.poster_path} alt={movie.title} />
+        <h3>{movie.title}</h3>
```

Comment:
Nice work on destructuring the movies. It will definately make this whole project a lot easier.